manala.influxdb/templates/config/base.conf.j2

```diff
@@ -197,12 +197,12 @@
   # Values without a size suffix are in bytes.
   {{ macros.config_row(config_data, 'max-index-log-file-size', '# max-index-log-file-size = "1m"', 0, true) }}
 
-  # The size of the internal cache used in the TSI index to store previously 
```

Comment:
Dont alter config template, its iso with upstream template, we need to keep extra spaces.